libcrux-ml-kem/Cargo.toml

```diff
@@ -27,6 +27,7 @@ libcrux-intrinsics = { version = "0.0.2-pre.2", path = "../libcrux-intrinsics" }
 # The hax config is set by the hax toolchain.
 [target.'cfg(hax)'.dependencies]
 hax-lib = { version = "0.1.0-pre.1", git = "https://github.com/hacspec/hax/" }
+hax-lib-macros = { version = "0.1.0-pre.1", git = "https://github.com/hacspec/hax/" }
```

Comment:
This shouldn't be needed. It is re-exported in the `hax-lib`.

Reply:
I've dropped the dependency. Had to change some paths because they were explicitly referring to `hax_lib_macros`.